tpm/commands.go

```diff
@@ -18,61 +18,23 @@ import (
 	"encoding/binary"
 	"errors"
 	"io"
-	"strconv"
+
+	"github.com/awly/go-tpm/tpmutil"
```

Comment:
Is it possible to make this github.com/google/go-tpm/tpmutil, since this PR will cause that directory to exist? The way I usually do this for testing is to check out my repo (in this case, it would be github.com/awly/go-tpm) into "${GOPATH}/src/github.com/google/go-tpm", even though it's really under awly. Then the go compiler is happy with the paths, and the checked-in version will work. @